.github/labels.yml

```diff
@@ -32,3 +29,30 @@
 - name: enhancement
   description: adds a new feature or improves on an existing one
   color: ad12cc
+- name: dependencies
+  description: Update one or more dependencies version
+  color: 8732bc
+- name: major
+  description: Increment the major version when merged
+  color: c5000b
+- name: minor
+  description: Increment the minor version when merged
+  color: f1a60e
+- name: patch
+  description: Increment the patch version when merged
+  color: 870048
+- name: release
+  description: Create a release when this pr is merged
+  color: 007f70
+- name: internal
+  description: Changes only affect the internal API
+  color: 696969
+- name: documentation
+  description: Changes only affect the documentation
+  color: cfd3d7
+- name: tests
+  description: Add or improve existing tests
+  color: ffd3cc
+- name: performance
+  description: Improve performance of an existing feature
+  color: f4b2d8
```

Comment:
![image](https://user-images.githubusercontent.com/5557458/95605940-8f6f5900-0a51-11eb-8ddb-a1afc121348e.png)
Can you tweak the colours a little, I would say the following are too close:
- bug <> minor
- cron <> release
- blocked <> patch

Reply:
Sure, they were just the defaults.

Reply:
I've updated, does it pull the labels from the default branch? i.e. next?

Reply:
No it runs the `sync-labels.yml` workflow on merge into release